pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1529,7 +1529,7 @@ assertNoAdditions {
   };
 
   intellitab-nvim = super.intellitab-nvim.overrideAttrs {
-    dependencies = [ self.nvim-treesitter ];
+    dependencies = [ self.nvim-treesitter-legacy ];
```

Comment:
Only uses `get_indent` from `nvim-treesitter` https://github.com/pta2002/intellitab.nvim/blob/main/lua/intellitab/init.lua#L29-L31 and that exists on `main` https://github.com/nvim-treesitter/nvim-treesitter/blob/4fc09bee78e91bf4ba471cdab4bf9dfa37fde51c/lua/nvim-treesitter/indent.lua#L112